resources/js/Pages/Admin/Users/ResetLinkModal.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <jet-confirmation-modal :show="user != null" @close="closeModal">
+    <template #title>
+      Generate reset link for {{ user ? user.name : "" }}
+    </template>
+
+    <template #content>
+      <!-- If the form is processing, display the loading screen -->
+      <div v-show="isProcessing">
+        <p>Loading...</p>
+      </div>
+      <!-- Else, Display the resetLink, if one is set-->
+      <div v-show="resetLink != null && !isProcessing">
+        <p>Reset link generated:</p>
+        <br/>
+        <pre class="text-sm overflow-x-auto whitespace-pre-wrap break-all">{{ resetLink }}</pre>
+      </div>
+      <!-- Else, Display the warning text before continuing -->
+      <div v-show="resetLink == null && !isProcessing">
+        <p>Warning: This will invalidate any existing password reset tokens of the user.</p>
+      </div>
```

Comment:
I suggest you use the actual v-if , v-else-if, v-else directives here. v-show will just set the display to none instead of conditionally rendering these items.